### PR TITLE
Adds committee-related multiactions

### LINKED ISSUE #150: Orga:Committees
- MultiActions
- Create/Update: no "meeting_as_template"

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Fixes #150

### CODE REVIEW

**Review by tsiegleauq — CHANGES_REQUESTED** [review]
Client cannot start. I am getting a white page with no errors and not logs. Master and every branch works fine. Restarted docker, `make stop-dev` etc. Changing to master works fine for loading OS4 changing into this branch breaks the client

**Comment by GabrielInTheWorld** [comment]
> Client cannot start. I am getting a white page with no errors and not logs. Master and every branch works fine. Restarted docker, `make stop-dev` etc. Changing to master works fine for loading OS4 changing into this branch breaks the client

Do you have the latest commits for every branch?

- auth-service: edfb247a2398ae69dd5c0ff44e7cdf692b7c1b80
- autoupdate-service: d1a51b9bfecb8880b85044f58613946414492d02
- backend: 3e449bc4a9987e35ebe366ec2790d88f5cc6dd05
- datastore-service: 49abad2adadcfd09c3467a0f0b9f581a127f1258
- manage-service: a40e5bd940c41a1eb98533a01f046c0061e2d866
- permission-service: 26f9fece39bf18933b3d9a838e323182542549aa

**Comment by tsiegleauq** [comment]
`git submodule status`

```
 edfb247a2398ae69dd5c0ff44e7cdf692b7c1b80 openslides-auth-service (remotes/origin/HEAD)
 d1a51b9bfecb8880b85044f58613946414492d02 openslides-autoupdate-service (v0.1.0-93-gd1a51b9)
 3e449bc4a9987e35ebe366ec2790d88f5cc6dd05 openslides-backend (remotes/origin/HEAD)
+a8a9c8b96c56ed4180d898dd602a7e1b2459aacb openslides-client (heads/master)
 49abad2adadcfd09c3467a0f0b9f581a127f1258 openslides-datastore-service (remotes/origin/HEAD)
+5d91cad7581b9bddbfe19ed43f8d2e6d5617c378 openslides-manage-service (remotes/origin/HEAD)
 6db3ecb09ecc4b49b20bb7fb6b24a057678ea27f openslides-media-service (heads/openslides4-dev)
+26f9fece39bf18933b3d9a838e323182542549aa openslides-permission-service (remotes/origin/HEAD)
```

**Comment by GabrielInTheWorld** [comment]
> Client cannot start. I am getting a white page with no errors and not logs. Master and every branch works fine. Restarted docker, `make stop-dev` etc. Changing to master works fine for loading OS4 changing into this branch breaks the client

I cannot reproduce your described behavior.

My steps:
1. On my new branch `orga-committees`
2. `make stop-dev`
3. `make run-dev`
4. Go to https://localhost:8000 -> sign in as `admin`
5. Reload the page (by `F5`)
6. Delete the default meeting
7. Reload the page (by `F5`)
8. Sign out and sign in again

Second test:
1. On branch `master`
2. `make stop-dev`
3. `make run-dev`
4. go to https://localhost:8000 -> sign in as `admin`
5. Reload the page (by `F5`)
6. Switch to my new branch `orga-committees`
7. Reload the page (by `F5`)
8. Sign out and sign in again

Everything works fine.

What are your steps?

**Comment by tsiegleauq** [comment]
The same. Tried again and had no luck, same results as before.
Clearing cache did not help, restarting the docker service did neither.
The client has no interesting docker output.

The client loads until the first router outlet and then stays blank.

**Comment by GabrielInTheWorld** [comment]
> > Client cannot start. I am getting a white page with no errors and not logs. Master and every branch works fine. Restarted docker, `make stop-dev` etc. Changing to master works fine for loading OS4 changing into this branch breaks the client
> 
> I cannot reproduce your described behavior.
> 
> My steps:
> 
> 1. On my new branch `orga-committees`
> 2. `make stop-dev`
> 3. `make run-dev`
> 4. Go to https://localhost:8000 -> sign in as `admin`
> 5. Reload the page (by `F5`)
> 6. Delete the default meeting
> 7. Reload the page (by `F5`)
> 8. Sign out and sign in again
> 
> Second test:
> 
> 1. On branch `master`
> 2. `make stop-dev`
> 3. `make run-dev`
> 4. go to https://localhost:8000 -> sign in as `admin`
> 5. Reload the page (by `F5`)
> 6. Switch to my new branch `orga-committees`
> 7. Reload the page (by `F5`)
> 8. Sign out and sign in again
> 
> Everything works fine.
> 
> What are your steps?

Ah, I think, I get the same behavior:
1. On my new branch `orga-committees`
2. On page https://localhost:8000/ (the dashboard)
3. `make stop-dev`
4. `make run-dev`
5. Reload -> an empty white page with no error message...

Please go manually to the login-page and retest again.

File: client/src/app/core/core-services/active-meeting-id.service.ts [hunk 1]
```diff
@@ -51,7 +51,6 @@ export class ActiveMeetingIdService {
     }
 
     public start(): void {
-        this._meetingId = 1;
```
Comment (tsiegleauq) [inline client/src/app/core/core-services/active-meeting-id.service.ts:54]: re-entering this line allows my client to load fine. removing this line breaks the client

**Comment by tsiegleauq** [comment]
> Ah, I think, I get the same behavior:
> 
> 1. On my new branch `orga-committees`
> 2. On page https://localhost:8000/ (the dashboard)
> 3. `make stop-dev`
> 4. `make run-dev`
> 5. Reload -> an empty white page with no error message...
> 
> Please go manually to the login-page and retest again.

That works. Perhaps the forwarding to /login is broken or somehow tied to the `_meetingId` in the `active-meeting-service`?

**Comment by GabrielInTheWorld** [comment]
> > Ah, I think, I get the same behavior:
> > 
> > 1. On my new branch `orga-committees`
> > 2. On page https://localhost:8000/ (the dashboard)
> > 3. `make stop-dev`
> > 4. `make run-dev`
> > 5. Reload -> an empty white page with no error message...
> > 
> > Please go manually to the login-page and retest again.
> 
> That works. Perhaps the forwarding to /login is broken or somehow tied to the `_meetingId` in the `active-meeting-service`?

Yeah, I think so, too. I will split this into another PR.

File: client/src/app/core/core-services/active-meeting-id.service.ts (hunk 1 above)
Comment (GabrielInTheWorld) [reply client/src/app/core/core-services/active-meeting-id.service.ts]: I've let this as it was before.

File: client/src/app/management/components/committee-list/committee-list.component.ts [hunk 2]
```diff
@@ -59,6 +63,23 @@ export class CommitteeListComponent extends BaseListViewComponent<ViewCommittee>
         this.router.navigate(['create'], { relativeTo: this.route });
     }
 
+    public async forwardToCommittees(): Promise<void> {
+        const content = this.translate.instant(
+            'This will add or remove the following groups for all selected participants:'
```
Comment (tsiegleauq) [inline client/src/app/management/components/committee-list/committee-list.component.ts:68]: Thats probably an artifact? Not sure what the correct hint would be

`Set or remove forwarding from the selected committees:`

or something

File: client/src/app/management/components/committee-list/committee-list.component.ts [hunk 3]
```diff
@@ -59,6 +63,23 @@ export class CommitteeListComponent extends BaseListViewComponent<ViewCommittee>
...
+        );
+        const ADD = _('Forward to committee(s)');
+        const REMOVE = _('Unforward to committee(s)');
+        const choices = [ADD, REMOVE];
+        const selectedChoice = await this.choiceService.open(content, this.repo.getViewModelList(), true, choices);
```
Comment (tsiegleauq) [inline client/src/app/management/components/committee-list/committee-list.component.ts:73]: story for another day, but I suppose we need to make the choice service prettier. Thats some weird buttons.

image

**Review by tsiegleauq — APPROVED** [review]
Fine I guess, @emanuelschuetze should probably look over the naming of things before merging.

**Pull request merged**